bundles/org.eclipse.osgi/supplement/src/org/eclipse/osgi/util/NLS.java

```diff
@@ -141,18 +145,50 @@ public static String bind(String message, Object[] bindings) {
 	 *
 	 * @param baseName the base name of a fully qualified message properties file.
 	 * @param clazz the class where the constants will exist
+	 * @param charset the charset for the given properties file
+	 * @since 3.22
```

Comment:
```suggestion
	 * @since 1.12
```

Reply:
Yes, that makes more sense. Eclipse complained I should use `3.22` instead. Noted.